README.md

```diff
@@ -1,25 +1,41 @@
 # IETF Multiformats Working Group
 
 Multiformats are a collection of self-describing data formats that consist of an
-inline data header and a data value. Multiformats have binary and text-based
-representations and are used to express base encodings (multibase),
-cryptographic hashes (multihash), cryptographic keys (multikey), network
-addresses (multiaddr), and a variety of other binary serialization formats
-(multicodec).
+inline data header and a data value. They are composable in that some of the
+more complex forms take another multiformat header/value tuple as value,
+recursively. Multiformats are designed to compose into binary streams with both
+header values and data values expressed in unsigned varints (a slight variation
+from protobuf varints and the UNIX LEB128 standard), with the exception of
+multibase, which base-encodes any valid binary multiformat and prepends its
+header to the target encoding rather than to the binary data value. The unsigned
+varint-based binary multiformats include headers for cryptographic hashes
+(multihash), cryptographic keys (multikey), network addresses (multiaddr), and a
+variety of other binary serialization formats that do not meet all the
+requirements to form part of the above registries (the rest of the
```

Comment:
This language is confusing and sounds scary. :)

I don't know what "other binary serialization formats" are and I don't know why they "do not meet all of the requirements"... and I don't know why a charter is talking about that sort of stuff. Sounds like you're describing stuff that's out of scope and then suggesting that the group should work to standardize those things?

Reply:
hehe good point.  upon inspection, almost every entry in multicodecs that is NOT part of one of the named specs/families/sub-registries is an "almost-entry", i.e. a non-cryptographic hash (which is thus not a multihash) or an addressing format that is not part of multiaddr.  these might never make it to "final" anyways and might stay experimental/tribal, so feel free to edit that out or counterpropose a less-scary way of describing the rest of multicodecs!

Reply:
Would [the most recent commit](https://github.com/msporny/charter-ietf-multiformats/pull/1/commits/fd23d36d3f7e9ad3b147f458f5575e152374d2d4) do the trick, or still too scary to ship? also happy to take counterproposals or tweaks, i'm just not sure how to honestly present the "loose entries" that aren't part of any of the multi*s.